### ISSUE TITLE
Dashboard failures, possibly due to incorrectly configured test platform

### ISSUE THREAD

**Issue by mhoemmen** [trilinos/Trilinos#1645]
@trilinos/framework @trilinos/tpetra 

ascic141, build name `Linux-gcc-4.9.3-Sierra_MPI_release_DEV_ETI_SERIAL-ON_OPENMP-OFF_PTHREAD-ON_CUDA-OFF_COMPLEX-ON`, has 21 Tpetra test failures.  Here is the latest list:

https://testing.sandia.gov/cdash/viewTest.php?onlyfailed&buildid=3072459

They all generate the following output:
```
p=0: *** Caught standard std::exception of type 'std::runtime_error' :
 
  Constructing View and initializing data with uninitialized execution space
  Traceback functionality not available
```
I was wondering at first whether the tests were using the default execution space instead of their Node's execution space.  However, check out the following failing test output:

https://testing.sandia.gov/cdash/testDetails.php?test=40862107&build=3072459

This is supposed to be for the default Node, yet the test is failing.  It looks like this could be a broken configuration.

**Comment by bartlettroscoe** [comment]
IWho owns this build?  In any case, it is listed as a "Specialized" build so it should not be spamming anyone with CDash emails.  And since it is a "Specialized" build, there are no offical policies for fixing these builds that I now of.  It is the wild west.

**Comment by mhoemmen** [comment]
@prwolfe or @rrdrake may need to know about this, but we can always file new issues as needed.  The fix for #1889 (currently blocked on the `Kokkos::is_initialized` feature) might also help.
